tests/integration/100.command.dev.test.js

```diff
@@ -252,6 +253,69 @@ test('Serves an Edge Function with a rewrite', async (t) => {
   })
 })
 
+test('Serves an Edge Function that includes context with site information', async (t) => {
+  await withSiteBuilder('site-with-fully-qualified-redirect-rule', async (builder) => {
```

Comment:
I mean something like this, since the site doesn't actually have a fully qualified redirect rule. (I notice we've been copy/pasting that test a lot, so the name is in a bunch of places incorrectly 🙈)

```suggestion
  await withSiteBuilder('site-with-edge-function-printing-site-info', async (builder) => {
```

Reply:
Ah yes I was wondering about that! Yeah I'd love to change that. Will do! Thanks